model.py

```diff
@@ -25,66 +25,57 @@ class Genre(db.Model):
 
     __tablename__ = "genres"
 
-    genre_id = db.Column(db.Integer,
-                        autoincrement=True,
-                        primary_key=True)
+    genre_id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(64), nullable=True)
 
     def __repr__(self):
         """Provide helpful representation when printed."""
 
         return f"<Genre genre_id={self.genre_id} name={self.name}>"
 
-# association_table = db.Table('association', 
-#     db.Column('location_id', db.Integer, db.ForeignKey('locations.location_id'), primary_key=True),
-#     db.Column('movie_id', db.Integer, db.ForeignKey('movies.movie_id'),primary_key=True),  
-# )
-#maybe explore this more later
+
+association_table = db.Table('movies_locations',
```

Comment:
You need this in order to simplify the logic in your routes. Take a look at the changes I made to your route for context on how this is being used.